internal/weapons/bow/compound/compound.go

```diff
@@ -72,6 +63,20 @@ func weapon(char core.Character, c *core.Core, r int, param map[string]int) stri
 		icd = c.F + cd
 		stackExpiry = c.F + stackDuration
 
+		char.AddMod(core.CharStatMod{
+			Key: "compoundbow",
+			Amount: func() ([]float64, bool) {
+				if c.F > stackExpiry {
+					stacks = 0
+				}
```

Comment:
Hmmm this method of setting stacks to 0 seems unsafe to me actually, as I think the `Amount` function isn't called if a mod is expired, so there's the risk of `stacks` remaining at 4 (or whatever number) forever basically.

I think the easiest solution to resolve this is to add a status whenever compound bow is proc'd, then change this line to reset stacks to 0 if the duration = 0.  Then there would be one other duration check right before the stack increment check to reset stacks to 0 which should cover all cases?

@srliao maybe I'm overcomplicating this though and it should just work?

Reply:
IMO this is fine since it doesn't matter if stacks remain at 4 as long as we don't add 4 stacks. However I meant to say that this check should be both here in the Amount call and also in the OnDamage call... since the OnDamage call needs to make sure it doesn't start off at 4 stacks when it should have reset. 

@jordanlovato Sorry about that. Misleading comment from me.